xopt/generators/bayesian/expected_improvement.py

```diff
@@ -50,7 +50,7 @@ def _get_acquisition(self, model):
         ].dropna()
         objective_data = self.vocs.objective_data(valid_data, "")
```

Comment:
please replace 
```
valid_data = self.data[
            pd.unique(self.vocs.variable_names + self.vocs.output_names)
        ].dropna()
objective_data = self.vocs.objective_data(valid_data, "")
```
with 
```
objective_data = self.vocs.objective_data(self.data, "").dropna().min()
```
which fixes the problem of independent nans for different objectives/constraints